### PR TITLE
 Changing name in package generates new package:  PR for #180 

### PR DESCRIPTION
**Pull request by infograf768**
This should solve https://github.com/joomla-projects/com_localise/issues/180 by deleting the older file when changing the name of the xml

### CODE REVIEW

File: component/admin/models/package.php [hunk 1]
```diff
@@ -313,6 +313,22 @@ public function getItem()
 	 */
 	public function save($data)
 	{
+		// When editing a package, find the original path
+		$app = JFactory::getApplication('administrator');
+		$this_id = $app->getUserState('com_localise.edit.package.id');
```
Comment (phproberto) [inline component/admin/models/package.php]: Can we call this `$id` following joomal standard?

File: component/admin/models/packagefile.php [hunk 2]
```diff
@@ -311,6 +311,22 @@ public function getItem()
 	 */
 	public function save($data)
 	{
+		// When editing a package, find the original path
+		$app = JFactory::getApplication('administrator');
+		$this_id = $app->getUserState('com_localise.edit.package.id');
+
+		if (!empty($this_id))
```
Comment (phproberto) [inline component/admin/models/packagefile.php]: Same here please `$id`

**Comment by infograf768** [comment]
Updated

**Comment by javigomez** [comment]
Works!

screen shot 2014-07-17 at 11 57 56

Thanks @infograf768 

This pull closes  #180

**Pull request merged**